### PR TITLE
Add ScaleStrategy.MaxUnavailable docs for CloneSet

### PR DESCRIPTION
**Pull request by veophi**
Signed-off-by: veophi <vec.g.sun@gmail.com>

Add ScaleStrategy.MaxUnavailable docs for CloneSet

### CODE REVIEW

File: docs/user-manuals/cloneset.md [hunk 1]
```diff
@@ -172,6 +172,29 @@ So there will be no limit to CloneSet name.
 
 Don't worry. Even if you enable the `CloneSetShortHash`, CloneSet will still recognize and manage the old Pods with full revision label.
 
+## Scale features
+
+### Scale up in streaming style
```
Comment (FillZpp) [inline docs/user-manuals/cloneset.md:177]: Scale up with rate limit

File: docs/user-manuals/cloneset.md [hunk 2]
```diff
@@ -172,6 +172,29 @@ So there will be no limit to CloneSet name.
...
+
+**FEATURE STATE:** Kruise v1.0.0
+
+Users can specify `ScaleStrategy. MaxUnavailable` to limit the step size of CloneSet **Scaling Up**, so as to minimize the impact on application services.
```
Comment (FillZpp) [inline docs/user-manuals/cloneset.md:181]: `ScaleStrategy.MaxUnavailable` -- no space in it.

Comment (veophi) [reply docs/user-manuals/cloneset.md]: done

File: docs/user-manuals/cloneset.md (hunk 1 above)
Comment (veophi) [reply docs/user-manuals/cloneset.md]: done

File: docs/user-manuals/cloneset.md [hunk 3]
```diff
@@ -172,6 +172,29 @@ So there will be no limit to CloneSet name.
...
+## Scale features
+
+### Scale up with rate limit
+
+**FEATURE STATE:** Kruise v1.0.0
+
+Users can specify `ScaleStrategy.MaxUnavailable` to limit the step size of CloneSet **Scaling Up**, so as to minimize the impact on application services.
+This value can be an absolute number (e.g., 5) or a percentage of desired number of Pods (e.g., 10%). Default value is `nil` (i.e., empty pointer), which indicates non-limitation.
+
+`ScaleStrategy. MaxUnavailable` field can cooperate with 'Spec.MinReadySeconds' field to work, for example:
```
Comment (FillZpp) [inline docs/user-manuals/cloneset.md:184]: `ScaleStrategy.MaxUnavailable`

'Spec.MinReadySeconds' should be `Spec. MinReadySeconds `

**Review by FillZpp — APPROVED** [review]
/lgtm

**Pull request merged**